Source/Urho3D/Graphics/OpenGL/OGLGraphics.cpp

```diff
@@ -945,16 +967,20 @@ void Graphics::DrawInstanced(PrimitiveType type, unsigned indexStart, unsigned i
     glDrawElementsInstancedANGLE(glPrimitiveType, indexCount, indexType, reinterpret_cast<const GLvoid*>(indexStart * indexSize),
         instanceCount);
 #else
+  #ifndef GL_ES_VERSION_3_0

```

Comment:
Preprocessor directives should be at the start of the line (in other places as well).